packages/p2p/src/components/buy-sell/no-ads/__tests__/no-ads.spec.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { createBrowserHistory } from 'history';
+import { Router } from 'react-router';
+import NoAds from '../no-ads';
+
+const mock_store_values = {
+    buy_sell_store: {
+        is_buy: true,
+        local_currencies: [
+            {
+                text: 'USD',
+                value: 'USD',
+                is_default: true,
+            },
+        ],
+        selected_local_currency: 'USD',
+        setCreateSellAdFromNoAds: jest.fn(),
+    },
+    general_store: {
+        handleTabClick: jest.fn(),
+        is_barred: false,
+    },
+    my_ads_store: {
+        setShowAdForm: jest.fn(),
+    },
+};
+
+jest.mock('Stores', () => ({
+    ...jest.requireActual('Stores'),
+    useStores: jest.fn(() => mock_store_values),
+}));
+
+jest.mock('@sendbird/chat', () => ({
```

Comment:
**Non-blocking:** Can we update jest.config instead? So we dont need to refactor these test files in the future 🙏